app/javascript/lib/scp-api-metrics.js

```diff
@@ -129,6 +129,14 @@ export function logSearch(type, searchParams, perfTime) {
   )
 }
 
+/**
+ * Log create user annotation metrics
+ */
+export function logCreateUserAnnotation() {
```

Comment:
Do we still need to report activity to Google Analytics?  I had thought we'd moved all metrics/activity reporting to Mixpanel.

Reply:
Yup, we want to preserve GA logging until we resolve SCP-2801, then remove all GA logging per SCP-2449.